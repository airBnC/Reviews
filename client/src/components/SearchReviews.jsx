import React from 'react';
import PropTypes from 'prop-types';

const SearchReviews = props => (
  <div> Search Reviews  </div>
);

export default SearchReviews;
