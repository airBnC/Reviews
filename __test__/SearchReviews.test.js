import React from 'react';
import { configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import SearchReviews from '../client/src/components/SearchReviews';

configure({ adapter: new Adapter() });

describe('SearchReviews', () => {
  it('should render correctly', () => {
    const output = shallow(<SearchReviews />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
