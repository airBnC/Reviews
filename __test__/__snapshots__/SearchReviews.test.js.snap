// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchReviews should render correctly 1`] = `
<div>
   Search Reviews  
</div>
`;
